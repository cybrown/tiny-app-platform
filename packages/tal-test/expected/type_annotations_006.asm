main()
  entry:
    DeclareLocal      name: toto1, mutable: false, hasInitialValue: false
    Pop               inBlock: false
    DeclareLocal      name: toto2, mutable: false, hasInitialValue: false
    Pop               inBlock: false
    DeclareLocal      name: string_split, mutable: false, hasInitialValue: false
    Pop               inBlock: false
    Local             string_split
