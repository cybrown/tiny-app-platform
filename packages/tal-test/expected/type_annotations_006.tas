let toto1: () => (string | number)
let toto2: () => string | number
let string_split: (a: string, b: string) => array<string>
string_split
