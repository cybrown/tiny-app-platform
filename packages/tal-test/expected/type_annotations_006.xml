<program>
  <declarelocal>
  </declarelocal>
  <declarelocal>
  </declarelocal>
  <declarelocal>
  </declarelocal>
  <local />
</program>
